core/Plugin/Manager.php

```diff
@@ -1076,11 +1076,20 @@ private function installPluginIfNecessary(Plugin $plugin)
 
         if ($saveConfig) {
             PiwikConfig::getInstance()->forceSave();
+            $this->clearCache($pluginName);
         }
     }
 
     public function isTrackerPlugin(Plugin $plugin)
     {
+        if (!$this->isPluginInstalled($plugin->getPluginName())) {
```

Comment:
We should not load a plugin in tracker mode that is not installed. Otherwise the plugin might wants to read or write to columns that do not exist yet